server/src/test/java/io/druid/segment/loading/CacheTestSegmentLoader.java

```diff
@@ -114,9 +115,7 @@ public void cleanup(DataSegment segment) throws SegmentLoadingException
   @Override
   public List<Path> getSegmentStorageLocations()
   {
-    List<Path> locationsList = Lists.newArrayList();
-    locationsList.add(Paths.get(locationDir.getAbsolutePath()));
-    return locationsList;
+    return locationDir.stream().map(File::toPath).collect(Collectors.toList());
```

Comment:
be careful with streams, I don't know that they are guaranteed to be closed.  I'm not sure if this needs to be closed or not, but if it does you need extra code to make sure it gets closed.